recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "keyrings.alt" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "2f1a55849504644795076291bcc66a7266e0388dea13fe23a89d25c99e355d22" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+  run:
+    - keyring
+    - python
+
+test:
+  imports:
+    - keyrings.alt
+
+about:
+  home: https://github.com/jaraco/keyrings.alt
+  license: MIT
```

Comment:
Ideally we would package the license file, but that doesn't seem to be included yet. Once that is the recipe should be updated to include this.

xref: https://github.com/jaraco/keyrings.alt/pull/8